src/number/n_Z.jl

```diff
@@ -102,6 +102,10 @@ function show(io::IO, c::Integers)
    print(io, "Integer Ring")
 end
 
+function AbstractAlgebra.expressify(n::n_Z; context = nothing)::Any
+  return AbstractAlgebra.expressify(BigInt(n), context = context)
```

Comment:
Codecov complains that this code (and some of the other code you add) is not covered by tests. Could you perhaps add some basic tests, to verify these new methods work (and keep working)?